test/tizen_capi/unittest_tizen_capi.cpp

```diff
@@ -509,16 +506,80 @@ TEST(nntrainer_capi_nnmodel, addLayer_04_p) {
   EXPECT_EQ(status, ML_ERROR_NONE);
 }
 
+TEST(nntrainer_capi_nnmodel, addLayer_05_n) {
```

Comment:
This test has same effect with addLayer_01_p

Reply:
okay